src/stores/app/index.js

```diff
@@ -176,7 +163,7 @@ export default class AppStore extends Store {
     if (get(this.createResult, 'app_id')) {
       this.createAppId = get(this.createResult, 'app_id');
       this.createStep = 3; //show application has been created page
-      this.updateMeunApps = true;
+      // this.updateMeunApps = true;
```

Comment:
The next line code: await this.fetchAll(); add parameter menuApps, because create app will change the menu apps

Reply:
will fix it